.github/workflows/ci.yml

```diff
@@ -77,8 +77,13 @@ jobs:
           DOMAIN: "http://127.0.0.1:8000"
           SECRET_KEY: "test"
           CSRF_SECRET: "test"
-          SESSION_SECRET_KEY: "test"
+          REFRESH_TOKEN_EXPIRE_MINUTES: 1440
           ACCESS_TOKEN_EXPIRE_MINUTES: 30
+          VERIFY_TOKEN_SECRET_KEY: secret1
+          ACCESS_TOKEN_SECRET_KEY: secret2
+          REFRESH_TOKEN_SECRET_KEY: secret3
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** CI workflow file contains hard-coded secrets.

Using hard-coded secrets in CI workflows can lead to security risks. Consider using GitHub Secrets or environment variables.